website/src/contribute/release-guide.md

```diff
@@ -428,20 +419,59 @@ Javadoc to the Javadoc for other modules that Beam depends on.
+* If the issue is a known flaky test, make an attempt to fix the issue. However, if the issue may take too long to fix (to the discretion of the release manager):
+  * Perform or delegate manual testing of flaky issue to ensure no release blocking issues.
+  * Update the `Fix Version` field to the version of the next release. Please consider discussing this with stakeholders and the dev@ mailing list, as appropriate.
+
+For all other JIRA issues:
+
+* If the issue has not been resolved and it is acceptable to defer this until the next release, update the `Fix Version` field to the new version you just created. Please consider discussing this with stakeholders and the dev@ mailing list, as appropriate.
+* If the issue has not been resolved and it is not acceptable to release until it is fixed, the release cannot proceed. Instead, work with the Beam community to resolve the issue.
+
+If there is a bug found in the RC creation process/tools, those issues should be considered high priority and fixed in 7 days.
+
+### Review Release Notes in JIRA
+
+JIRA automatically generates Release Notes based on the `Fix Version` field applied to issues. Release Notes are intended for Beam users (not Beam committers/contributors). You should ensure that Release Notes are informative and useful.
+
+Open the release notes from the [version status page](https://issues.apache.org/jira/browse/BEAM/?selectedTab=com.atlassian.jira.jira-projects-plugin:versions-panel) by choosing the release underway and clicking Release Notes.
+
+You should verify that the issues listed automatically by JIRA are appropriate to appear in the Release Notes. Specifically, issues should:
+
+* Be appropriately classified as `Bug`, `New Feature`, `Improvement`, etc.
+* Represent noteworthy user-facing changes, such as new functionality, backward-incompatible API changes, or performance improvements.
+* Have occurred since the previous release; an issue that was introduced and fixed between releases should not appear in the Release Notes.
+* Have an issue title that makes sense when read on its own.
+
+Adjust any of the above properties to the improve clarity and presentation of the Release Notes.
 
 ### Checklist to proceed to the next step
 
 1. Release Manager’s GPG key is published to `dist.apache.org`
-2. Release Manager’s GPG key is configured in `git` configuration
-3. Release Manager has `org.apache.beam` listed under `Staging Profiles` in Nexus
-4. Release Manager’s Nexus User Token is configured in `settings.xml`
-5. JIRA release item for the subsequent release has been created
-6. There are no release blocking JIRA issues
-7. Release Notes in JIRA have been audited and adjusted
-8. Combined javadoc has the appropriate contents.
-9. Release branch has been created
-10. Originating branch has the version information updated to the new version
-11. Nightly snapshot is in progress (do revisit it continually)
+1. Release Manager’s GPG key is configured in `git` configuration
```

Comment:
Please don't do this. The point of markdown is that it is readable in its unrendered form.

Reply:
Gotcha, reverted back to explicit numbering